packages/kit/src/runtime/server/endpoint.js

```diff
@@ -10,6 +10,10 @@ function error(body) {
 	};
 }
 
```

Comment:
Adding a jsdoc here (`/** @param {string} s */`) removes the linting error

Reply:
thanks